pose_xyt_t pose
pose_xyt_t parent_pose
float64 weight
